docs/BUILDING_DOCKER.md

```diff
@@ -6,22 +6,12 @@ To use Docker, you'll need either Docker Desktop or Docker Toolbox installed and
 
 You'll also need to prepare a local version of the project with a copied base ROM (see steps [2](../README.md#2-clone-the-repository) and [3](../README.md#3-prepare-a-base-rom) of the Linux instructions).
 
-## 2. Create the Docker image
+## 2. Create and start the Docker image build
 
-From inside your local project, run the following command:
+From the root of your local project, run the following command:
 
 ```bash
-docker build . -t oot
+docker-compose up --build
 ```
 
-## 3. Start the container
-
-To start the container, you can mount your local filesystem into the Docker container and run an interactive bash session.
-
-```bash
-docker run -it --rm --mount type=bind,source="$(pwd)",destination=/oot oot /bin/bash
-```
```

Comment:
I'm not too well-versed with docker, is there still a good way to get an interactive shell into the container after building? That seems necessary if you want to use this build method for development.

I suppose that's one "benefit" of having the user run `make setup` and `make` manually, since they'll probably want to do that anyway during development.

Reply:
https://github.com/ptweezy/oot/pull/1

This would keep the image open after build to allow you to shell into it right after build. What do you think?

Reply:
hmm seems a bit hacky, what about this instead? https://stackoverflow.com/a/55953120

also could you put a command to open a shell in BUILDING_DOCKER.md too?

Reply:
@cadmic are you referring to the 'tty: true' option? I do have that enabled in the docker-compose.yml already, but for some reason on my machine, I have to have some blocking command. '/bin/bash' works, as well as 'tail -f /dev/null', but it does not open an interactive shell without taking the extra step to shell into the pod. My experience on Windows + Docker is the same as this guy in the same exact thread: https://stackoverflow.com/a/71425623

Reply:
alright nvm, I guess `tail -f /dev/null` is fine

Reply:
Added the command! :)